templates/.eslintignore

```diff
@@ -1,3 +1,12 @@
+# Ignore everything in the root directory except for package.json
+/*
```

Comment:
This line may cause ESLint to ignore all the files, not only the files in the root directory.
![image](https://user-images.githubusercontent.com/45935381/236718436-d1e65118-0f53-4286-8d95-c6ee46345c55.png)